conf/environments.js

```diff
@@ -102,6 +102,11 @@ module.exports = {
             ecmaVersion: 6
         }
     },
+    es7: {
```

Comment:
If they're not consistently present, won't that be confusing in the future when I can set `env` to 6, 8, and 11, say, but not to 7, 9, and 10?